docs/pages/_app.js

```diff
@@ -41,7 +41,10 @@ ponyfillGlobal.muiDocConfig = {
   csbIncludePeerDependencies: (deps, { versions }) => {
     const newDeps = { ...deps };
 
-    newDeps['@mui/material'] = versions['@mui/material'];
+    // #default-branch-switch
+    // Check which version of `@mui/material` should be resolved when opening docs examples in StackBlitz or CodeSandbox
+    newDeps['@mui/material'] =
+      versions['@mui/material'] !== 'next' ? versions['@mui/material'] : 'latest';
```

Comment:
I've gone with a solution, which still allows monorepo to provide a specific `@mui/material` version, but if you think that we should just go for simplicity—we can do that. 👌 

```diff
-versions['@mui/material'] !== 'next' ? versions['@mui/material'] : 'latest';
+'latest';
```